src/app/components/show-players/show-players.component.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+function ShowPlayersController($element, SavedPlayers, SelectedPlayers, PlayersFactory) {
```

Comment:
Per la definizione di un componente ti suggerirei il seguente approccio come gia fatto per gli altri file:
- definizione in un file
- template file esterno
- controller file esterno